src/tools/build-manifest/src/main.rs

```diff
@@ -425,16 +439,15 @@ impl Builder {
         assert!(t!(child.wait()).success());
     }
 
-    fn write_manifest(&self, manifest: &str, name: &str) {
-        let dst = self.output.join(name);
-        t!(t!(File::create(&dst)).write_all(manifest.as_bytes()));
-        self.hash(&dst);
-        self.sign(&dst);
+    fn write_channel_files(&self, channel_name: &str, manifest: &Manifest) {
+        self.write(&toml::to_string(&manifest).unwrap(), channel_name, ".toml");
+        self.write(&manifest.date, channel_name, "-date.txt");
+        self.write(&manifest.git_commit_hash, channel_name, "-git-commit-hash.txt");
```

Comment:
Do we for sure need a dedicated file for the git commit? This'll already be present in the manifest, and the `date.txt` was only added much later as a request (ideally this extra file wouldn't exist)

Reply:
We don’t really need it, just like we don’t really need `date.txt`. A separate file makes it marginally easier for Servo’s bootstrap to extract the commit hash, but we can also parse the TOML file.

I can remove it if you prefer.